packages/google-cloud-access-context-manager/noxfile.py

```diff
@@ -458,3 +468,70 @@ def prerelease_deps(session, protobuf_implementation):
             "PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION": protobuf_implementation,
         },
     )
+
+
+@nox.session(python="3.13")
+@nox.parametrize(
+    "protobuf_implementation",
+    ["python", "upb"],
+)
+def core_deps_from_source(session, protobuf_implementation):
+    """Run all tests with local versions of core dependencies installed,
+    rather than pulling core dependencies from PyPI.
+    """
+
+    # Install all dependencies
+    session.install(".")
```

Comment:
Do you want a `-e` here like in the previous test?

Reply:
Fixed in https://github.com/googleapis/google-cloud-python/pull/13637/commits/eeb2efd3ee59ed691e071ead09b4effffd30395b. https://github.com/googleapis/synthtool/pull/2071 will take care of updating this in all packages.